### ISSUE TITLE
```CalendarEventDraft``` API for setting link is missing. 

### ISSUE THREAD

**Issue by gtalha07** [acterglobal/a3#727]

**Comment by kumarpalsinh25** [comment]
@gtalha07 
Is this issue is still persists? Can you check and confirm?

**Comment by gtalha07** [comment]
Seeing ```CalendarEventDraft``` struct in rust. We still don't support it. The question remains, do we still need this ? It makes sense for virtual location event where link is required. @gnunicorn

**Comment by gnunicorn** [comment]
yes, "link" isn't really that clear. If it is part of a location, then it does make sense. If that's what you mean.
